src/esp/assets/ResourceManager.cpp

```diff
@@ -461,8 +461,10 @@ std::string ResourceManager::createColorMaterial(
     gfx::PhongMaterialData::uptr phongMaterial =
         gfx::PhongMaterialData::create_unique();
     phongMaterial->ambientColor = materialColor.ambientColor;
-    phongMaterial->diffuseColor = materialColor.diffuseColor;
-    phongMaterial->specularColor = materialColor.specularColor;
+    // NOTE: This multiplication is a hack to roughly balance the Phong and PBR
+    // light intensity reactions.
+    phongMaterial->diffuseColor = materialColor.diffuseColor * 0.175;
+    phongMaterial->specularColor = materialColor.specularColor * 0.175;
```

Comment:
Hm, wouldn't it make sense to do this in the PBR shader directly instead? Like, the will go through many other changes for IBL anyway, and that would keep the hack in a single place.

Reply:
Yeah, agreed. It makes sense to scale up in the PBR intesnties or scale down the Phong intensities in the shader, but I needed to get something in for the release visuals.